### PR TITLE
cmd: add `config` command to change config of a volume

### LINKED ISSUE #1063: Find a better way to change volume quota
Currently `format` is used to update the volume quota, which is not proper

### PR DESCRIPTION
**Pull request by SandyXSD**
close #1063 

For now the `format` command is not touched, so it can still be used to change the config as well.  However, the new `config` is a recommended way, and `format` will be deprecated (only keep the real **FORMAT** function) after one or two minor versions.

### CODE REVIEW

File: cmd/config.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+	}
+	m := meta.NewClient(ctx.Args().Get(0), &meta.Config{Retries: 10, Strict: true})
+
+	format, err := m.Load()
+	if err != nil {
+		return err
+	}
+	var msg strings.Builder
+	for _, flag := range ctx.LocalFlagNames() {
+		switch flag {
```
Comment (davies) [inline cmd/config.go:65]: ignore the default value

Comment (SandyXSD) [reply cmd/config.go:65]: `LocalFlagNames` returns only flags explicitly specified in the command

File: cmd/config.go [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+		case "capacity":
+			msg.WriteString(fmt.Sprintf("%10s: %d -> %d\n", flag, format.Capacity, ctx.Uint64(flag)))
```
Comment (davies) [inline cmd/config.go:67]: Show the message when it's different?

File: cmd/config.go [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+		Name:  "capacity",
+		Value: 0,
+		Usage: "the limit for space in GiB",
+	},
+	&cli.Uint64Flag{
+		Name:  "inodes",
+		Value: 0,
+		Usage: "the limit for number of inodes",
+	},
+	&cli.StringFlag{
```
Comment (davies) [inline cmd/config.go:37]: We should have `--bucket` here to update the endpoint

File: cmd/config.go [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+		Name:  "access-key",
+		Usage: "Access key for object storage (env ACCESS_KEY)",
```
Comment (davies) [inline cmd/config.go:39]: Should we take the ACCESS_KEY from env?

User may not want to expose the secret in command line history

File: cmd/config.go (hunk 2 above)
Comment (SandyXSD) [reply cmd/config.go:67]: again the message shows up only if user input the option explicitly, so imo checking whether they're different is trivial

**Review by davies — APPROVED** [review]

**Review by davies — APPROVED** [review]

**Pull request merged**